Resources/input/fsh/ISiKVersicherungsverhaeltnisGesetzlich.fsh

```diff
@@ -9,10 +9,17 @@ Description: "Dieses Profil ermöglicht die Darstellung eines gesetzlichen Versi
   * ^slicing.discriminator.path = "$this"
   * ^slicing.rules = #open
   * ^short = "Primärer Identifier der Versicherung"
-  * ^definition = "Ein gesetzliches Versicherungsverhältnis sollte stets durch die eindeutige 30-stellige Versichertennummer identifiziert werden. Ist diese nicht bekannt, so wird die 10-stellige VersichertenID statt dessen verwendet."
-* identifier contains KrankenversichertenID 1..1 MS
+  * ^definition = "Ein gesetzliches Versicherungsverhältnis sollte stets durch die eindeutige 30-stellige Versicherungsnummer identifiziert werden. Ist diese nicht bekannt, so kann die 10-stellige KrankenversichertenID statt dessen verwendet werden. Diese ist aber nur in Verbindung mit dem Payor eindeutig einem Versicherungsverhältnis zuordenbar"
```

Comment:
@f-peverali sehe ich es richtig, dass kein Identifier aus den oben definierten nun MS ist? oder MUSS einer der beiden doch unterstützt werden? wenn ja, dann sollten wir das in der Spec so explizit formulieren - am besten sogar mit einem Profil-Constraint. Falls KVID-10 nur in Verbindung mit einem Payor verwendet werden darf, dann sollte m.M.n. auch hier ein Constraint formuliert werden. 

Für Testing stellt sich für mich in dem Context die Frage, ob man in der Vorbedingung die Unterstützung von einem der beiden Identifier fordern darf - die sind ja jetzt kein MS mehr...

Reply:
@patrick-werner , @alexzautke was haltet ihr von dem Constraint?
Ich würde hier auch nochmal an der Formulierung pfeilen,  um die Anforderung als solche klarer hervorzuheben.
Danke @alexey-tschudnowsky für den Hinweis!

Reply:
@alexey-tschudnowsky Ja, die MS-Vorgaben für Identifier haben wir jetzt komplett entfernt (entsprechend der Diskussion aus dem letzten Industrieworkshop). Die Slices sind nur vorhanden falls ein System unbedingt hier einen Identifier hinterlegen möchte, dann gibt es wenigstens grobe Vorgaben dafür.

Solange wir kein gutes FHIR-natives Verzeichnis für Versicherungen haben ist payor.reference schwierig zu bedienen. Das Beste wird auf absehbare Zeit dort .display bleiben. Das Feld dort ist bereits ein Pflichtfeld.

Reply:
Ggf. trotzdem eine SOll Anforderung für die KVID und Payor-reference (S.u.)?

Reply:
@f-peverali Auf was würde die Referenz zeigen? Es ist nicht sinnvoll, dass jedes KIS die Stammdaten für Krankenversicherungen pflegen vorhalten muss um die Organization befüllen zu können.